.github/workflows/ci.yml

```
@@ -0,0 +1,168 @@
+        python -m py_compile src/madspark/agents/*.py
+    
+    - name: Validate project structure
+      run: |
+        echo "Checking project structure..."
+        test -f README.md || (echo "Missing README.md" && exit 1)
+        test -f config/requirements.txt || (echo "Missing config/requirements.txt" && exit 1)
+        test -d src/madspark || (echo "Missing src/madspark directory" && exit 1)
+        test -f src/madspark/__init__.py || (echo "Missing src/madspark/__init__.py" && exit 1)
+        test -d src/madspark/agents || (echo "Missing src/madspark/agents directory" && exit 1)
+        test -f src/madspark/agents/__init__.py || (echo "Missing src/madspark/agents/__init__.py" && exit 1)
+        echo "✅ Project structure validation passed"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)